src/app/pages/item/ItemList.tsx

```diff
@@ -1,36 +1,27 @@
 import React from "react"
-import PropTypes from "prop-types"
 import { connect } from "react-redux"
-import { refreshList } from "./Item.actions"
+import { refreshList } from "../../redux/Item/Item.actions"
+import { ItemProps, Item } from "../../redux/Item/Item.types"
 
-const ItemList = ({ items, refreshList }) => {
+const ItemList = ({ items, refreshList }: ItemProps) => {
     console.log(items)
     return (
         <div>
             <button onClick={refreshList}>Refresh</button>
             <ul>
-                {items.map((item) => (
+                {items.map((item: Item) => (
                     <li key={item.id}>{item.name}</li>
                 ))}
             </ul>
         </div>
     )
 }
 
-const mapStateToProps = (state) => ({
+const mapStateToProps = (state: any) => ({
```

Comment:
Should have `RootState` interface/type (a union of all reducers type).
https://redux.js.org/recipes/usage-with-typescript#typing-the-connect-higher-order-component